PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/sdk/StatusRequester.java

```diff
@@ -26,67 +25,74 @@ public class StatusRequester {
     public static void makeRequest(@Nullable SdkInitializationListener initializationListener) {
         listener = initializationListener;
 
-        String url = PrebidMobile.getPrebidServerHost().getHostUrl();
-        if (url.contains("/openrtb2/auction")) {
-            String statusUrl = url.replace("/openrtb2/auction", "/status");
-            ServerConnection.fireWithResult(
-                statusUrl,
-                getResponseHandler()
-            );
+        String statusUrl;
+
+        String customStatusEndpointUrl = PrebidMobile.getCustomStatusEndpoint();
+        if (customStatusEndpointUrl != null) {
+            statusUrl = customStatusEndpointUrl;
         } else {
-            LogUtil.info("SDK doesn't support host urls without `/openrtb2/auction` part for now");
-            postOnMainThread(StatusRequester::onInitializationCompleted);
+            String url = PrebidMobile.getPrebidServerHost().getHostUrl();
+            if (url.contains("/openrtb2/auction")) {
+                statusUrl = url.replace("/openrtb2/auction", "/status");
+            } else {
+                LogUtil.info("SDK doesn't support host urls without `/openrtb2/auction`. Please set /status endpoint url using PrebidMobile.setCustomStatusEndpoint().");
+                postOnMainThread(StatusRequester::onInitializationCompleted);
+                return;
+            }
         }
+
+        ServerConnection.fireStatusRequest(
+            statusUrl,
+            getResponseHandler()
+        );
     }
 
     private static ResponseHandler getResponseHandler() {
         return new ResponseHandler() {
             @Override
             public void onResponse(BaseNetworkTask.GetUrlResult response) {
-                if (response.statusCode == 200) {
-                    try {
-                        JSONObject responseJson = new JSONObject(response.responseString);
-                        JSONObject applicationJson = responseJson.optJSONObject("application");
-                        if (applicationJson != null) {
-                            onInitializationCompleted();
-                            return;
-                        }
-                    } catch (JSONException exception) {
-                        onInitializationFailed("Wrong `/status` response: " + exception.getMessage());
-                        return;
-                    }
+                int statusCode = response.statusCode;
+                if (statusCode >= 200 && statusCode < 300) {
```

Comment:
Can we make this check the part of `BaseNetworkTask`. In order to reuse it in either places like:

```
response.isOkStatusCode
```